posts/2018-02-09-serverless-and-graphql-in-love.md

```diff
@@ -0,0 +1,608 @@
+---
+title: Running a scalable & reliable GraphQL endpoint with Serverless  
+description: Learn how to create GraphQL endpoints using Serverless Framework, Lambda, DynamoDB, RDS and a REST backend. First of a series of technical posts on Serverless + GraphQL.   
+date: 2018-02-13  
+layout: Post  
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/graphql.jpeg  
+authors:  
+  - SiddharthGupta
+---
+
+![Serverless and GraphQL family](https://user-images.githubusercontent.com/1587005/36154091-2d1a9bc0-1085-11e8-8280-183f25ebc080.png "Serverless and GraphQL family")
```

Comment:
@andrea-p the image is quite large. Do you think we can get it across with we just make everything smaller? :)

Reply:
as long as its visible and pretty and fits with the content ;)